modules/mediasquareBidAdapter.js

```diff
@@ -109,25 +113,30 @@ export const spec = {
      * @param {ServerResponse[]} serverResponses List of server's responses.
      * @return {UserSync[]} The user syncs which should be dropped.
      */
-  getUserSyncs: function(syncOptions, serverResponses, gdprConsent) {
+  getUserSyncs: function(syncOptions, serverResponses, gdprConsent, uspConsent) {
     let params = '';
     let endpoint = document.location.search.match(/msq_test=true/) ? BIDDER_URL_TEST : BIDDER_URL_PROD;
-    if (gdprConsent && typeof gdprConsent.consentString === 'string') {
-      if (typeof gdprConsent.gdprApplies === 'boolean') { params += `&gdpr=${Number(gdprConsent.gdprApplies)}&gdpr_consent=${gdprConsent.consentString}`; } else { params += `&gdpr_consent=${gdprConsent.consentString}`; }
-    }
-    if (syncOptions.iframeEnabled) {
-      return {
-        type: 'iframe',
-        url: endpoint + BIDDER_ENDPOINT_SYNC + '?type=iframe' + params
-      };
-    }
-    if (syncOptions.pixelEnabled) {
-      return {
-        type: 'image',
-        url: endpoint + BIDDER_ENDPOINT_SYNC + '?type=pixel' + params
-      };
+    if (serverResponses[0].body.hasOwnProperty('cookies') && typeof serverResponses[0].body.cookies === 'object') {
+      return serverResponses[0].body.cookies;
+    } else {
+      if (gdprConsent && typeof gdprConsent.consentString === 'string') {
+        if (typeof gdprConsent.gdprApplies === 'boolean') { params += `&gdpr=${Number(gdprConsent.gdprApplies)}&gdpr_consent=${gdprConsent.consentString}`; } else { params += `&gdpr_consent=${gdprConsent.consentString}`; }
+      }
+      if (uspConsent && typeof uspConsent === 'string') { params += '&uspConsent=' + uspConsent }
+      if (syncOptions.iframeEnabled) {
```

Comment:
`iframeEnabled` was deprecated in Prebid 3.0. Please refer to section 2 (userSync API) of this page: http://prebid.org/blog/pbjs-3

Reply:
I wasn't aware as a lot of modules still use it ^^ I actually don't really need it anymore so I just removed this check

Reply:
@matthieularere-msq 

Sorry, my mistake. Those settings are still being passed into the adapters as `syncOptions.iframeEnabled` and `syncOptions.pixelEnabled`. The change was to how they are passed in from the page in `setConfig`.

If you do need those `if` blocks, you can add them back in.

Reply:
No problem. As I said I don't really need it anymore so it's ok the way it is. Thanks